HW9/task1.py

```diff
@@ -0,0 +1,58 @@
+# Напишите следующие функции:
+# Нахождение корней квадратного уравнения
+# Генерация csv файла с тремя случайными числами в каждой строке. 100-1000 строк.
+# Декоратор, запускающий функцию нахождения корней квадратного уравнения с каждой тройкой чисел из csv файла.
+# Декоратор, сохраняющий переданные параметры и результаты работы функции в json файл
+import csv
+import random
+import math
+from typing import Callable
+
+NUMBER_OF_ROWS = 100
+MIN_NUMBER = -100
+MAX_NUMBER = 100
+NUMBER_OF_NUMBERS = 3
+
+
+def write_csv_nums(name: str):
+    with open(name, 'w', encoding='utf-8', newline='') as f_write:
+        csv_write = csv.writer(f_write)
+        for _ in range(0, NUMBER_OF_ROWS):
+            res2 = [random.randint(MIN_NUMBER, MAX_NUMBER) for _ in range(0, NUMBER_OF_NUMBERS)] 
```

Comment:
Забыл исключить 0 из генерации чисел:
```suggestion
            res2 = [random.randint(MIN_NUMBER, MAX_NUMBER) for _ in range(0, NUMBER_OF_NUMBERS) if _ != 0] 
```